tests/lean/non_exhaustive_error.lean.expected.out

```diff
@@ -16,3 +16,6 @@ f (nat.succ _) _
 g 0 (nat.succ _)
 g (nat.succ _) _
 non_exhaustive_error.lean:17:0: warning: declaration '_mutual.f.g' uses sorry
+non_exhaustive_error.lean:27:4: error: non-exhaustive match, the following cases are missing:
+const_name (term.mk _ (term.mk _ _ :: _))
+non_exhaustive_error.lean:27:0: warning: declaration 'const_name' uses sorry
```

Comment:
It looks right to me.  This is the same expression with more parens:
```
const_name (term.mk _ ((term.mk _ _) :: _))
```

I'm using the normal pretty-printer here, so it should not be buggier than the usual error messages.